configuration.sh

```diff
@@ -33,7 +33,8 @@ SENDER_EMAIL_PASS='passwd'
 SENDER_EMAIL='your.email@gmail.com'
 
 # Advanced configuration.
-CORE_DEVELOPMENT=false
```

Comment:
Do not push it into Github, CORE_DEVELOPMENT must stay to false in the master branch so that people installing Cytomine have a production-ready installer